src/main/java/org/mskcc/limsrest/service/GetSampleManifestTask.java

```diff
@@ -9,8 +9,15 @@
 import com.velox.sloan.cmo.recmodels.KAPALibPlateSetupProtocol1Model;
 import com.velox.sloan.cmo.recmodels.SampleModel;
 import com.velox.sloan.cmo.recmodels.SeqAnalysisSampleQCModel;
+import java.rmi.RemoteException;
+import java.text.SimpleDateFormat;
+import java.util.*;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.mskcc.cmo.shared.Library;
```

Comment:
importing these classes from the `cmo-metadb-shared-entities` repository now with jitpack